<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>238</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>368</width>
    <height>238</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>368</width>
    <height>238</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Toir Loader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/images/images/toirOff.png</normaloff>:/images/images/toirOff.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget #widget {
	background-color: #212121;
	border-radius: 10px;
}

#centralwidget #shadowImg {
	background-image: url(:/images/images/shadow.png);
	background-position: center;
	background-repeat: no-repeat;
}

#loadBtn {
	background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.511, y2:1, stop:0 rgba(212, 0, 255, 255), stop:1 rgba(155, 0, 194, 255));
	border-radius: 5px;
	color: #212121;
}
#loadBtn:hover {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(212, 212, 212, 255));
	border: 2px solid #d400ff;
	color: #d400ff;
}
#loadBtn:pressed {
	background-color: #212121;
	border: 2px solid #d400ff;
	color: #d400ff;
}

#footer QPushButton {
	color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.511, y2:1, stop:0 rgba(212, 0, 255, 255), stop:1 rgba(155, 0, 194, 255));
	background-color: #212121;
	border-radius: 18px;
}
#footer QPushButton:hover {
	color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(212, 212, 212, 255));
	background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.511, y2:1, stop:0 rgba(212, 0, 255, 255), stop:1 rgba(155, 0, 194, 255));
}
#footer QPushButton:pressed {
	color: #212121;
}

#statusLabel {
	color: #ffffff;
}

#titleBar QPushButton {
	background-color: transparent;
	border: 0;
}

#titleBar {
	background-color: #313131;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
}

#closeBtn {
	image: url(:/purpleIcons/images/purpleIcons/x.svg);
}
#closeBtn:hover {
	image: url(:/whiteIcons/images/whiteIcons/x.svg);
}

#minimizeBtn {
	image: url(:/purpleIcons/images/purpleIcons/minimize-2.svg);
}
#minimizeBtn:hover {
	image: url(:/whiteIcons/images/whiteIcons/minimize-2.svg);
}

#configBtn {
	image: url(:/purpleIcons/images/purpleIcons/settings.svg);
}
#configBtn:hover {
	image: url(:/whiteIcons/images/whiteIcons/settings.svg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QFrame" name="shadowImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>368</width>
      <height>238</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>330</width>
       <height>200</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QFrame" name="titleBar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources/res.qrc">:/images/images/toirOff.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tileTitle">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Toir Loader&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight|Qt::AlignVCenter">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="configBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="minimizeBtn">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="header">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="statusLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Toir is not open!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="body" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="loadBtn">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LOAD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="footer">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="footerLinks">
           <property name="spacing">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="discBtn">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Social Media Circled</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="ttBtn">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Social Media Circled</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="gitBtn">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Social Media Circled</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>)</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="ppBtn">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Social Media Circled</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
